### PR TITLE
Enhance/add jobs

### LINKED ISSUE #7: Add jobs to the orb
> The one thing I think you're missing currently is a few jobs. You've defined a lot of commands, and showed how to string them together in your examples, however you do not provide any jobs that would allow users to easily call a collection of commands in a predefined order. That would also allow you to make use of the executor you've defined.

https://circleci.com/docs/2.0/reusing-config/#jobs-defined-in-an-orb

### PR DESCRIPTION
**Pull request by speeddragon**
Fix #7

### CODE REVIEW

File: src/jobs/buikd-and-test.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+description: Build and test elixir source code
+executor: default
```
Comment (tmpduarte) [inline src/jobs/buikd-and-test.yml]: is the filename right: `buikd`?

Comment (speeddragon) [reply src/jobs/buikd-and-test.yml]: Fixed

File: src/jobs/build-and-deploy.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+description: Build and deploy elixir source code
+executor: default
+parameters:
+  branch:
+    description: Branch that to be deployed
```
Comment (tmpduarte) [inline src/jobs/build-and-deploy.yml]:
```suggestion
    description: Branch to be deployed
```

File: src/jobs/build-and-test.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
+description: Build and test elixir source code
+executor: default
+parameters:
+  coveralls:
+    description: Run tests using excoveralls for line coverage
```
Comment (tmpduarte) [inline src/jobs/build-and-test.yml]:
```suggestion
    description: Run tests using excoveralls for code coverage
```

**Review by tmpduarte — APPROVED** [review]

**Pull request merged**